### PR TITLE
Joshd/sc 94517/gh actions add support for tags in create

### PR DESCRIPTION
**Pull request by jdewinne**
Allow tags to be specified for create cluster

### CODE REVIEW

File: create-cluster/action.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ inputs:
     description: 'Time to wait for the cluster to have a status of `running`'
     required: false
     default: '20'
+  tags:
+    description: 'Tags to assign to the cluster.'
```
Comment (emosbaugh) [inline create-cluster/action.yml:35]: should this give a hint to the format?

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: create-cluster/action.yml (hunk 1 above)
Comment (jdewinne) [reply create-cluster/action.yml:35]: I think `description` is single line string. I just added an example in
example-workflows/development-helm.yaml

Comment (emosbaugh) [reply create-cluster/action.yml:35]: perhaps you should consider a different format if its too hard to document in a single line. how about comma separated key=value?

this action has good descriptions to use as an example

https://github.com/peter-evans/create-pull-request/blob/main/action.yml

Comment (jdewinne) [reply create-cluster/action.yml]: I guess, multiline strings also work for descriptions.

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**